modules/ROOT/pages/proc_uprgading_pipelines_operator.adoc

```diff
@@ -0,0 +1,11 @@
+[id="installing-the-pipelines-operator_{context}"]
+= Upgrading the Pipelines Operator to the newer version
```

Comment:
@boczkowska since there are no actionable instructions for the user, can we simply add this to the existing note under step 5 in the Installation Procedure? Without instructions we should not be calling a module a procedural module. Once you do that please remove the include in the Assembly too.